department-of-veterans-affairs/va.gov-cms
# VACMS-2934: Add html cleanup to table field.

## Description
See #2934 

## Testing done
Visual / behat

## QA steps
- As an administrator, go to /node/471/edit
- In Main content section, add a table paragraph
- Enter content with broken html tags in multiple cells
- Save form, and confirm table displays properly on node save
- Return to edit form, and confirm broken html has been cured in table cells
